build-conf/zCEE3.properties

```diff
@@ -0,0 +1,25 @@
+# Releng properties used by language/zCEE3.groovy
+
+#
+# Comma separated list of required build properties for zCEE3.groovy
+zCEE_requiredBuildProperties=zCEE_alwaysRebuildWAR, zCEE_shellEnvironment,\
+                                zCEE_gradlePath, zCEE_gradle_JAVA_OPTS
+
+
+#
+# When set to true, always rebuild the WAR file, even if one is found in the build folder of the z/OS Connect project folder.
+# When set to false, will pick the existing WAR file if found or will build it if no WAR file is found
```

Comment:
A WAR file should not be under version control at all. I think it is safe to assume, that we always need to produce a new *war file ZCEE3 anyway.

My suggestion would be to drop this property, and we always build when the openapi specification has changed.

Reply:
I completely agree, but it's the z/OS Connect Designer that creates the WAR file, and store it inside the Git repository (and there is nothing you can do to avoid it as far as I know). 
I think we should still propose this option, but set it as optional, and set the default to true (as I did in this version).

Reply:
Can we make a comment about using `.gitignore` to avoid storing the WAR there?